lib/search_solr_tools/harvesters/base.rb

```diff
@@ -124,7 +169,17 @@ def get_serialized_doc(doc, content_type)
         end
       end
 
-      # Get results from some ISO end point specified in the query string
+      # Return a string to help identify a document.  Can be overridden in
+      # child classes if the docs for the harvester are not in the "standard"
+      # hash format
+      def doc_identifier(doc)
```

Comment:
Where is this document hash format defined? I don't see this method used anywhere. I recommend leaving out the complexity at this point.